include/mlir/Dialect/Linalg/Transforms/LinalgTransformPatterns.td

```diff
@@ -24,6 +24,8 @@
 
 include "mlir/Dialect/Linalg/IR/LinalgOps.td"
 include "mlir/Dialect/Linalg/IR/LinalgLibraryOps.td"
+include "mlir/Dialect/AffineOps/AffineOps.td"
+include "mlir/Dialect/AffineOps/AffineOpsBase.td"
```

Comment:
AffineOps should realistically give you AffineOpsBase already.

Reply:
You are right, thank you!

Reply:
I still see it here, merge issue?

Reply:
Likely a merge issue. Removed again. Thank you.